exercises/list-ops/list_ops_test.rb

```diff
@@ -0,0 +1,107 @@
+gem 'minitest', '>= 5.0.0'
+require 'minitest/autorun'
+require_relative 'list_ops'
+
+class ListOpsTest < Minitest::Test
+  def test_count_empty
```

Comment:
So, people trying to solve this exercise will not be able to see the `example.rb` file, so the tests should be the guide here. So maybe we could call this `test_count_empty_array_elements`?  This way people know exactly what is expected just by reading the title.